netbox/dcim/svg/cables.py

```diff
@@ -34,9 +34,13 @@ class Node(Hyperlink):
         radius: Box corner radius, for rounded corners (default: 10)
     """
 
-    def __init__(self, position, width, url, color, labels, radius=10, **extra):
+    object = None
+
+    def __init__(self, position, width, url, color, labels, radius=10, object=object, **extra):
         super(Node, self).__init__(href=url, target='_parent', **extra)
 
+        self.object = object
```

Comment:
What's going on here? Why is the object needed?

Reply:
It is referenced by lines 379/387 and 401/402